### PR TITLE
Update base-it.yaml

### PR DESCRIPTION
**Pull request by Doc-Ninja**
Completed the italian translation.
SHOULD be free of typos
Made some changes to some choices made by other translator to make everything uniform (mainly fixing stuff having different names in different sections or changing the name of some machines to be feminine to flow better)
Notable things to check:
Line 329 Not sure if ":" breaks the sintax
Line 412 Should purple be changed to magenta?
Line 526 Not completely satisfied with this, might need rephrasing

### CODE REVIEW

File: translations/base-it.yaml [hunk 1]
```diff
@@ -31,75 +31,77 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+
```
Comment (DerFeldspatz) [inline translations/base-it.yaml]: one line too much here

File: translations/base-it.yaml [hunk 2]
```diff
@@ -387,50 +388,50 @@ ingame:
...
-                This is <strong>NOT</strong> an idle game! Build more extractors and belts to finish the goal quicker.<br><br>Tip: Hold <strong>SHIFT</strong> to place multiple extractors, and use <strong>R</strong> to rotate them.
+                Questo <strong>NON</strong> è un idle game! Costruisci più estrattori e nastri per raggiungere l'obiettivo più velocemente.<br><br>Suggerimento: Tieni premuto <strong>MAIUSC</strong> per piazzare estrattori multipli, e usa <strong>R</strong> per ruotarli.
 
     colors:
         red: Rosso
         green: Verde
         blue: Blu
         yellow: Giallo
-        purple: Viola
+        purple: Viola #shouldn't be "Magenta"?
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:413]: I would use one correct translation for `purple` and delete the comment. If it's incorrect you or someone else can still change it later.

File: translations/base-it.yaml [hunk 3]
```diff
@@ -502,181 +504,181 @@ buildings:
...
             description: >-
-                Changes how sensitive the zoom is (Either mouse wheel or trackpad).
+                Determina quanto è sensibile lo zoom (Sia rotella del mouse che trackpad).
             sensitivity:
-                super_slow: Super slow
-                slow: Slow
-                regular: Regular
-                fast: Fast
-                super_fast: Super fast
+                ssuper_slow: Molto lento
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:668]: Don't change the key

File: translations/base-it.yaml [hunk 4]
```diff
@@ -813,50 +811,48 @@ keybindings:
 
         rotateWhilePlacing: Ruota
         rotateInverseModifier: >-
-            Modifier: Rotate CCW instead
+            Modifier: Ruota in senso antiorario
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:815]: `Modifier` can be translated here

File: translations/base-it.yaml [hunk 5]
```diff
@@ -723,79 +725,75 @@ settings:
...
-                different building types.
+                Ogni tipo di edificio ricorderà l'ultimo orientamento che hai selezionato indipendentemente dagli altri tipi.
+                È utile se cambi spesso tipo di edificio.
 
 keybindings:
-    title: Keybindings
+    title: Comandi
     hint: >-
-        Tip: Be sure to make use of CTRL, SHIFT and ALT! They enable different placement options.
+        Tip: Usa spesso CTRL, MAIUSC e ALT! Abilitano opzioni di posizionamento alternative.
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:771]: `Tip`can be translated here

File: translations/base-it.yaml [hunk 6]
```diff
@@ -723,79 +725,75 @@ settings:
...
-                disabled: Disabled
+                one_minute: 1 Minuto
+                two_minutes: 2 Minuti
+                five_minutes: 5 Minuti
+                ten_minutes: 10 Minuti
+                twenty_minutes: 20 Minuti
+                disabled: Disabilitato
         compactBuildingInfo:
-            title: Compact Building Infos
+            title: Info edifici compatte.
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:751]: No punctuations in title

File: translations/base-it.yaml [hunk 7]
```diff
@@ -317,47 +319,46 @@ ingame:
...
-            tiles: <x> titles
+            tiles: <x> caselle
 
     # The notification when completing a level
     levelCompleteNotification:
         # <level> is replaced by the actual level, so this gets 'Level 03' for example.
         levelTitle: Livello <level>
         completed: Completato
-        unlockText: Sbloccato <reward>!
+        unlockText: Sbloccato: <reward>!
```
Comment (DerFeldspatz) [inline translations/base-it.yaml]: Change it to two lines and 'tab' the second one:
`unlockText: >-`
`    Sbloccato: <reward>!`

File: translations/base-it.yaml [hunk 8]
```diff
@@ -317,47 +319,47 @@ ingame:
...
-        unlockText: Sbloccato <reward>!
+        unlockText: >-
+            Sbloccato: <reward>!
         buttonNextLevel: Prossimo livello
 
     # Notifications on the lower right
     notifications:
-        newUpgrade: U  nuovo aggiornamento è disponibile!
-        gameSaved: La tua partita è stata salvata.
-
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:334]: You deleted a line too much

File: translations/base-it.yaml [hunk 9]
```diff
@@ -473,27 +475,28 @@ buildings:
...
-            name: &cutter Cutter
-            description: Cuts shapes from top to bottom and outputs both halfs. <strong>If you use only one part, be sure to destroy the other part or it will stall!</strong>
+            name: &cutter Tagliatrice
+            description: Taglia le forme verticalmente e restituisce le metà destra e sinistra. <strong>Se usi solo una parte, distruggi l'altra o la macchina si fermerà!</strong>
         quad:
-            name: Cutter (Quad)
-            description: Cuts shapes into four parts. <strong>If you use only one part, be sure to destroy the other part or it will stall!</strong>
+            name: Tagliatrice (4x)
+            description: Taglia le forme in quattro parti. <strong>Se usi solo una parte, distruggi le altre o la macchina si fermerà!
+</strong>
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:483]: Put it back to the line above

File: translations/base-it.yaml [hunk 10]
```diff
@@ -723,79 +726,75 @@ settings:
                 super_fast: Molto veloce
                 extremely_fast: Velocissimo
         enableTunnelSmartplace:
-            title: Smart Tunnels
+            title: Tunnel intelligenti
             description: >-
-                When enabled, placing tunnels will automatically remove unnecessary belts.
-                This also enables to drag tunnels and excess tunnels will get removed.
+                Se abilitato, piazzare un tunnel rimuoverà anche qualsiasi nastro superfluo.
+                Consente anche di trascinare i tunnel: i tunnel in eccesso verrano rimossi.
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:732]: 1 trailing space

File: translations/base-it.yaml [hunk 11]
```diff
@@ -723,79 +726,75 @@ settings:
...
-                to read.
+                Abilita la vignettatura che scurisce gli angoli dello schermo e rende i testi più leggibili.
 
         autosaveInterval:
-            title: Autosave Interval
+            title: Intervallo salvataggio automatico
             description: >-
-                Controls how often the game saves automatically. You can also disable it
-                entirely here.
+                Determina ogni quanto il gioco salva automaticamente.
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:741]: 1 trailing space

File: translations/base-it.yaml (hunk 2 above)
Comment (Doc-Ninja) [reply translations/base-it.yaml:413]: purple is "Viola", but I thought it was supposed to be changed to magenta

Comment (DerFeldspatz) [reply translations/base-it.yaml:413]: Then it's magenta. Did not know about that. It's only the key that did not get changed yet

File: translations/base-it.yaml [hunk 12]
```diff
@@ -317,47 +318,48 @@ ingame:
...
         buttonNextLevel: Prossimo livello
 
     # Notifications on the lower right
     notifications:
-        newUpgrade: U  nuovo aggiornamento è disponibile!
-        gameSaved: La tua partita è stata salvata.
-
+        newUpgrade: È disponibile un nuovo aggiornamento!
+        gameSaved: Partita salvata.
+
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:336]: Almost done. Only these trailing spaces left 😅

File: translations/base-it.yaml [hunk 13]
```diff
@@ -502,181 +504,182 @@ buildings:
...
         default:
             name: &trash Cestino
-            description: Accetta ingressi da tutti i lati e li distrugge. per sempre.
+            description: Accetta oggetti da tutti i lati e li distrugge. Per sempre.
 
         storage:
-            name: Storage
-            description: Stores excess items, up to a given capacity. Can be used as an overflow gate.
+            name: Stoccaggio
+            description: Immagazzina gli oggetti in eccesso, fino ad una certa capacità. Può essere usato come varco per le eccedenze #Literal translation, quite ugly, might need rephrasing
```
Comment (DerFeldspatz) [inline translations/base-it.yaml:526]: I'm not sure whether your comment will be visible ingame or not. Might wait for approval of @ tobspr

Comment (Doc-Ninja) [reply translations/base-it.yaml:526]: Don't know either, I am not familiar at all with Yaml

Comment (tobspr) [reply translations/base-it.yaml:526]: It will I *think*, I'll remove it

**Comment by tobspr** [comment]
Thanks!

**Pull request merged**